lib/daru/view/adapters/googlecharts.rb

```diff
@@ -47,10 +41,23 @@ def init_table(data=[], options={})
           # }
           # then directly DatTable is created using options. Use data=[] or nil
           @table = GoogleVisualr::DataTable.new(options)
+          @table.data = data
           add_data_in_table(data)
           @table
         end
 
+        def get_table(data)
+          if data.is_a?(Daru::View::Table) && data.table.is_a?(GoogleVisualr::DataTable)
+            data.table
+          elsif data.is_a?(GoogleVisualr::DataTable)
+            data
+          elsif data.is_a?(String)
```

Comment:
Check whether it is a valid URL or not. If not rescue the Exception + document this enhancement.

Reply:
Where should I write the documentation?

Reply:
[here](https://github.com/SciRuby/daru-view/pull/88/files#diff-cd0c826056999c96e0390d69150cc07bR18)